packages/bot-web-ui/src/components/load-modal/__tests__/recent.spec.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { mockStore, StoreProvider } from '@deriv/stores';
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
 import { mock_ws } from 'Utils/mock';
 import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
 import Recent from '../recent';
+import userEvent from '@testing-library/user-event';
```

Comment:
it looks like in our package `bot-web-ui` "eslint-plugin-simple-import-sort" doesn't work anymore to sort imports in the correct order...let's check it

Reply:
@maryia-matskevich-deriv i think we had removed this. 
Since there were test cases breaking in the packages because of the sorting.